src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -3327,7 +3327,7 @@ CHIP_ERROR DoorLockServer::sendClusterResponse(chip::app::CommandHandler * comma
     }
     else
     {
-        err = commandObj->AddStatus(commandPath, ToInteractionModelStatus(status));
+        err = commandObj->FailableAddStatus(commandPath, ToInteractionModelStatus(status));
```

Comment:
More fake error handling: nothing ever checks the return value of sendClusterResponse....

Reply:
Updated this to return void.